workflowtrigger/db/redis_client.go

```diff
@@ -301,6 +304,20 @@ func (r *RedisClientImpl) FetchTransactionless(key string, objectType ObjectType
 	panic(errors.New("objectType did not match type"))
 }
 
+func (r *RedisClientImpl) DeleteByPrefix(prefix string) error {
+	keys, err := r.client.Do("KEYS", fmt.Sprintf("%s*", prefix))
```

Comment:
Add KEYS and DEL as a RedisCommands and use the variables here instead of explicit strings. Also use the redisWrapperFunc method to when making Redis commands, it will help catch errors (won't need to return an error for the method either).

Reply:
Refactored `DeleteByPrefix` method